ledger/src/blockstore_db.rs

```diff
@@ -1287,6 +1287,11 @@ impl Database {
     pub fn live_files_metadata(&self) -> Result<Vec<LiveFile>> {
         self.backend.live_files_metadata()
     }
+
+    pub fn compact_range_cf<C: Column + ColumnName>(&self, from: &[u8], to: &[u8]) {
```

Comment:
I'm ok leaving this open - it is a layer down after all and not on `Blockstore`.

Additionally, I had pondered running a compaction on blockstore of warehouse nodes before we load them up to google bucket so this is a nice kick to run that test soon